docs/_data/components/schemas/forecast-limit-period.yaml

```diff
@@ -21,21 +21,38 @@ forecast-period-proposal:
 
   allOf:
     - $ref: '#/period'
-    - $ref: ./limit-data.yaml
+    - $ref: ./limit-proposal.yaml
 
-forecast-period-snapshot-slim:
+proposal-considered:
+  type: object
   allOf:
-    - $ref: '#/forecast-period-proposal'
+    - $ref: '#/proposal'
+    - type: object
+      properties:
+        source:
+          $ref: ./data-provenance.yaml
+        proposal-disposition:
+          type: string
+          enum:
+            - Used
+            - Rejected
+      required:
+        - proposal-disposition
+
+snapshot-slim:
+  allOf:
+    - $ref: '#/period'
+    - $ref: ./limit-data.yaml
     - type: object
       properties:
-        updated-time:
+        updated-time: #TODO: remove this property???
```

Comment:
@getorymckeag lmk what you think, but I believe this redundant with the provenance, i.e., I assume the use case for this was making sure that the limit being used is actually new. I think that the right question is which proposals were used since if you're using new proposals you get a new limit.

Reply:
Sure